### ISSUE TITLE
请问用的是UER的哪一个初始化模型？

### ISSUE THREAD

**Issue by crazycv** [thinkingmanyangyang/smp-ewect-code#1]
你好，想要问一下你们使用的是uer的哪一个预训练模型？

**Comment by thinkingmanyangyang** [comment]
uer mixed corpus large



---原始邮件---
发件人: "crazycv"<notifications@github.com&gt;
发送时间: 2020年10月23日(周五) 下午3:02
收件人: "thinkingmanyangyang/smp-ewect-code"<smp-ewect-code@noreply.github.com&gt;;
抄送: "Subscribed"<subscribed@noreply.github.com&gt;;
主题: [thinkingmanyangyang/smp-ewect-code] 请问用的是UER的哪一个初始化模型？ (#1)




 
你好，想要问一下你们使用的是uer的哪一个预训练模型？

**Comment by ML-PPAP** [comment]
logits = torch.softmax(logits, dim=-1)
请问一下既然是多分类任务，为什么这里使用的是softmax而不是sigmoid
![image](https://user-images.githubusercontent.com/49780566/113987216-bbdd2400-9880-11eb-8fe4-592f35f65781.png)

**Comment by thinkingmanyangyang** [comment]
这个比赛是单分类任务



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年4月8日(周四) 下午3:41
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [thinkingmanyangyang/smp-ewect-code] 请问用的是UER的哪一个初始化模型？ (#1)




 
logits = torch.softmax(logits, dim=-1)
 请问一下既然是多分类任务，为什么这里使用的是softmax而不是sigmoid
